src/routes/volume/VolumeBulkActions.js

```diff
@@ -160,6 +163,7 @@ function bulkActions({ selectedRows, engineImages, bulkDeleteVolume, showBulkEng
     { key: 'expandVolume', name: 'Expand Volume', disabled() { return selectedRows.length === 0 || selectedRows.some((item) => item.state !== 'detached') || !conditionsScheduled() } },
     { key: 'updateBulkReplicaCount', name: 'Update Replicas Count', disabled() { return selectedRows.length === 0 || isHasStandy() || disableUpdateBulkReplicaCount() || upgradingEngine() } },
     { key: 'updateBulkDataLocality', name: 'Update Data Locality', disabled() { return selectedRows.length === 0 || isHasStandy() || disableUpdateBulkDataLocality() || upgradingEngine() } },
+    { key: 'updateSnapshotDataIntegrity', name: 'Update Snapshot Data Integrity', disabled() { return selectedRows.length === 0 || selectedRows.some((item) => item.state !== 'attached') } },
```

Comment:
Why not `selectedRows.some((item) => attachable(item))`? I assume all volumes should be detached when updating this bulk setting?

Reply:
done